src/components/Chat/hooks/useCreateChatClient.ts

```diff
@@ -29,7 +33,7 @@ export const useCreateChatClient = <SCG extends ExtendableGenerics = DefaultGene
   }
 
   useEffect(() => {
-    const client = new StreamChat<SCG>(apiKey);
+    const client = new StreamChat<SCG>(apiKey, undefined, options);
```

Comment:
```suggestion
    const client = new StreamChat<SCG>(apiKey, options);
```

What's the `undefined` for? 🤔 

![image](https://github.com/user-attachments/assets/f66d42aa-4e27-4430-b643-8edc4c4e4133)

Reply:
Oh - I see, please ignore. :)

Reply:
TS not being happy about the possibility that the 2nd parameter can be a string. This way it can be sure, the 3rd param is `StreamChatOptions | undefined`.